docs/site/pages/dsl/index.mdx

```diff
@@ -0,0 +1,152 @@
+---
+title: DSL Content Overview
+---
+
+# TSX/JSX Content Authoring (Player DSL)
+
+While Player content _can_ be written directly in JSON, it's definitely not the preferable authoring format. To take advantage of existing developer tool-chains, Player provides a mechanism for authoring content in (J/T)SX as React components and simple TypeScript objects. The Player CLI can then be used to transpile the React tree into a JSON content.
+
+## Writing DSL Content
+
+In order to use the DSL to write content, your plugin library should ship a DSL component package. These will define the primitive _components_ to use to build up the tree. Authorship of these components is covered in the [Writing DSL Components](../assets/dsl) secton. The Player Reference Assets ship their own DSL Components via the `@player-ui/reference-assets-components` pacakge.
+
+In the examples below, we will use the Player Reference Assets Components.
+
+### Basic Setup 
+
+To get started, you'll need the following dependencies in your `package.json`:
+
+```json
+"dependencies": {
+    "@player-tools/dsl": "0.4.1",
+    "@player-tools/cli": "0.4.1",
+    "@player-ui/reference-assets-components": "0.6.0",
+    "@types/react": "17.0.39",
+    "react": "17.0.2"
+}
+```
+
+Next, you'll need to configure your environment for DSL Compilation and JSON validation. Below is a basic configuration that can be added in your `package.json`. For a more detailed explination and examples on further customization please refer to the [CLI](../tools/cli) section. 
+
+```json
+"player": {
+    "dsl": {
+      "src": "./src/main/tsx",
+      "outDir": "./out"
+    },
+    "json": {
+      "src": "./out/*.json"
+    },
+  }
+```
```

Comment:
Does this block blow up in the preview since it's not valid JSON due to the missing surrounding `{}`?

Reply:
Oh maybe, let me fix that